test/system/211-dbus.bats

```diff
@@ -0,0 +1,80 @@
+# shellcheck shell=bats
+#
+# Copyright © 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load 'libs/bats-support/load'
+load 'libs/bats-assert/load'
+load 'libs/helpers'
+
+setup() {
+  bats_require_minimum_version 1.7.0
+  _setup_environment
+  cleanup_containers
+}
+
+teardown() {
+  cleanup_containers
+}
+
+@test "dbus: session bus" {
+  local expected_response
+  expected_response="$(gdbus call --session --dest org.freedesktop.DBus
+   --object-path /org/freedesktop/DBus --method org.freedesktop.DBus.Peer.Ping)"
```

Comment:
You need to use a backslash (ie., `\`) when wrapping lines.

Currently the tests are failing with:
```
fedora-38 | not ok 264 dbus: session bus in 180ms
fedora-38 | # (in test file test/system/211-dbus.bats, line 35)
fedora-38 | #   `--object-path /org/freedesktop/DBus --method org.freedesktop.DBus.Peer.Ping)"' failed with status 127
fedora-38 | # Error: Object path is not specified
fedora-38 | # /home/zuul-worker/src/github.com/containers/toolbox/test/system/211-dbus.bats: line 36: --object-path: command not found
fedora-38 | not ok 265 dbus: system bus in 183ms
fedora-38 | # (in test file test/system/211-dbus.bats, line 60)
fedora-38 | #   `org.freedesktop.systemd1.Manager Version)"' failed with status 127
fedora-38 | # Error: Object path is not specified
fedora-38 | # /home/zuul-worker/src/github.com/containers/toolbox/test/system/211-dbus.bats: line 61: --object-path: command not found
fedora-38 | # /home/zuul-worker/src/github.com/containers/toolbox/test/system/211-dbus.bats: line 62: org.freedesktop.systemd1.Manager: command not found
```

Reply:
Fixed.